app/assets/v2/js/grants/fund.js

```diff
@@ -83,6 +85,33 @@ $(document).ready(function() {
             $('#sub_new_approve_tx_id').val(transactionHash);
             const linkURL = etherscan_tx_url(transactionHash);
 
+            let data = {
+              'contributor_address': $('#contributor_address').val(),
+              'amount_per_period': $('#amount').val(),
+              'real_period_seconds': realPeriodSeconds,
+              'frequency': $('#frequency_count').val(),
+              'frequency_unit': $('#frequency_unit').val(),
+              'token_address': $('#js-token').val(),
+              'token_symbol': $('#token_symbol').val(),
+              'gas_price': $('#gas_price').val(),
+              'sub_new_approve_tx_id': transactionHash,
+              'num_tx_approved': $('#period').val(),
+              'network': $('#network').val(),
+              'csrfmiddlewaretoken': $("#js-fundGrant input[name='csrfmiddlewaretoken']").val()
+            };
+
+            $.ajax({
+              type: 'post',
+              url: '',
+              data: data,
+              success: function(json) {
+                console.log('successfully saved subscription');
```

Comment:
shouldnt we do more than console log here?

Reply:
I agree, it would be better not just have a log, but we also dont want to take any action. We just want to next ajax to fire. Do you have any suggestions for what to add here? @owocki?